code/datums/components/riding/riding_mob.dm

```diff
@@ -335,6 +335,9 @@
 
 /datum/component/riding/creature/cow/handle_specials()
 	. = ..()
+//MONKESTATION EDIT START
+	vehicle_move_delay = 1.25
```

Comment:
i really dislike these being hardcoded tbh

Reply:
Removed them for now, gonna bring it back in a future PR just need time to think about better way to add